app/src/features/Batches/BatchListItem.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+
+import { Link, Paper, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { Replay } from "@material-ui/icons";
+import moment from "moment";
+import { useTranslation } from "react-i18next";
+
+import Button from "common/components/Button";
+import { Batch } from "services/api/generated/api.generated";
+
+import { KIBANA_URL } from "../../constants";
+import BatchCancel from "./BatchCancel";
+
+type BatchListItemType = {
+  batch: Batch;
+};
+
+const useStyles = makeStyles((theme) => ({
+  listItem: {
+    marginBottom: theme.spacing(0.5),
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: theme.spacing(1.3, 2, 1, 2),
+    flexWrap: "wrap",
+  },
+  listItemActions: {
+    display: "flex",
+    alignItems: "center",
+    flexWrap: "wrap",
+  },
+  margin: {
+    marginRight: theme.spacing(1),
+  },
+  title: {
+    display: "flex",
+  },
+}));
+
+const createKibanaLink = (batchCreation: string, batchEnd: string) => {
+  return `${KIBANA_URL}/app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'${batchCreation}',to:'${batchEnd}'))&_a=(columns:!(_source),filters:!(),interval:auto,query:(language:kuery,query:error),sort:!())`;
+};
```

Comment:
have you tested this url ? does it return any result ?

Reply:
Yes ! It worked when I tried

Reply:
`batchEnd` must be optional and default to "now" if not specified